### ISSUE TITLE
[Qt Style Sheet Highlighter] Dynamic Properties break parsing completly

### ISSUE THREAD

**Issue by std-any-emplace** [Quick-Turn-Studio/CLionSupportForQt#22]
This snippet is accepted by Qt5.1x, but breaks parsing in Qt Style Sheet Highlighter completely at `=true`. `="true"` is accepted.
```
QPushButton[prop=true]
{
	background-color: red;
	text-align: center;
	qproperty-flat: false;
}
```

**Comment by grabusr** [comment]
Hi,
Version 0.1.8 is ready to download. Please let's us know if it solves your problems.

**Comment by std-any-emplace** [comment]
Problem fixed in 0.1.8.
